src/fsharp/pars.fsy

```diff
@@ -2672,6 +2674,7 @@ rawConstant:
   | BIGNUM { SynConst.UserNum $1 } 
   | stringOrKeywordString { SynConst.String ($1,lhs parseState) } 
   | BYTEARRAY { SynConst.Bytes ($1,lhs parseState) }
+  | STRING_INTERPOLATED { SynConst.InterpolatedString ($1,lhs parseState) }
```

Comment:
What happens if interpolated strings are used in patterns - i.e. do we get a good error saying this is not supported?